src/libostree/ostree-impl-system-generator.c

```diff
@@ -109,6 +109,11 @@ require_internal_units (const char *normal_dir, const char *early_dir, const cha
       < 0)
     return glnx_throw_errno_prefix (error, "symlinkat");
 
+  if (symlinkat (SYSTEM_DATA_UNIT_PATH "/ostree-relabel-etc.service", normal_dir_dfd,
```

Comment:
I'm OK with this as is, but just a note: I think we can change things so that the initramfs code itself acts as a generator and writes the enablement symlink into `/run` directly then.

As is it's a bit silly because the initramfs writes the file in `/run` which says that transient etc is enabled, then we boot in the real root, run a binary which reads that file and exits if the flag isn't found...but we can bypass all that and be slightly more efficient by just telling systemd whether or not to run the binary.